tests/test_query_info.py

```diff
@@ -0,0 +1,76 @@
+from contextlib import contextmanager
+
+from clickhouse_driver import errors
+from tests.testcase import BaseTestCase
+
+
+class QueryInfoTestCase(BaseTestCase):
+
+    @property
+    def sample_query(self):
+        return 'SELECT * FROM test ORDER BY foo DESC LIMIT 5'
+
+    @contextmanager
+    def sample_table(self):
+        with self.create_table('foo UInt8'):
+            self.client.execute('INSERT INTO test (foo) VALUES',
+                                [(i,) for i in range(42)])
+            self.client.reset_last_query()
```

Comment:
Why do we need to `reset_last_query` after insert?

Reply:
I did not want `self.client` to have any previous queries in test cases to potentially mess with assertions.